src/json-patch.cpp

```diff
@@ -59,7 +69,7 @@ void json_patch::validateJsonPatch(json const &patch)
 
 		try {
 			// try parse to path
-			[[maybe_unused]] const auto p = json::json_pointer{op["path"].get<std::string>()};
+			JSV_MAYBE_UNUSED const auto p = json::json_pointer{op["path"].get<std::string>()};
```

Comment:
What about using the old-fashioned `(void)`-cast to eliminate this warning?

Reply:
~~I think it's only a warning when the compiler supports this attribute =?~~

Playing with this on [compiler explorer](https://godbolt.org/z/iYJywL) it seems compilers with less then C++17 STL support (which this feature is apart of) compile but still produce a warning regardless.

Reply:
```suggestion
			const auto p = json::json_pointer{op["path"].get<std::string>()}; p;
```

Reply:
And without using a variable at all?

Reply:
.... brillant !

Reply:
```suggestion
			json::json_pointer{op["path"].get<std::string>()};
```

Reply:
Yeah, but does it still do the job - ie. doesn't the compiler throw away the code because of not using the instance.

Reply:
Sadly yes =?